source/unified-test-format/unified-test-format.rst

```diff
@@ -1253,6 +1253,10 @@ The structure of this object is as follows:
   - ``commandName``: Optional string. Test runners MUST assert that the command
     name matches this value.
 
+  - ``databaseName``: Optional string. Test runners MUST assert that the
+    database name matches this value. The YAML file SHOULD use an `alias node`_
+    for this value (e.g. ``databaseName: *database0Name``).
```

Comment:
Note that will should be accompanied by a new changelog entry that not only summarizes the changes but also indicates the schema version bump (see existing entries for prior art).